src/base/bittorrent/sessionimpl.cpp

```diff
@@ -580,6 +580,13 @@ SessionImpl::SessionImpl(QObject *parent)
 
 SessionImpl::~SessionImpl()
 {
+    if (m_torrentsQueueChanged)
```

Comment:
Maybe it is better to move https://github.com/qbittorrent/qBittorrent/blob/f6735401f4738bac6dce92ed569e49873edbf5f9/src/base/bittorrent/sessionimpl.cpp#L2907-L2908
to above this `if` block

Reply:
> Maybe it is better to move

👍 
Done.